ydb/public/api/grpc/ydb_tablet_v1.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package Ydb.Tablet.V1;
+option java_package = "com.yandex.ydb.tablet.v1";
+
+import "ydb/public/api/protos/ydb_tablet.proto";
+
+service TabletService {
```

Comment:
I recommend adding a comment, that this service is a very low entity and is used for database hacking.

Reply:
Done